lib/index.js

```diff
@@ -1,7 +1,8 @@
 exports.AccessManager = require('./AccessManager').default;
 exports.Client = require('./Client').default;
 exports.Channel = require('./Channel').default;
-exports.UserInfo = require('./UserInfo').default;
+exports.User = require('./User').default;
```

Comment:
Fixed in: https://github.com/ccm-innovation/react-native-twilio-chat/pull/25/commits/9a62714ee9f8c2a48be22f03ab7ad912adf1012a